src/actions/headerActions.js

```diff
@@ -0,0 +1,6 @@
+export const ADD_FILE = 'ADD_FILE';
+
```

Comment:
redux actions/reducers, etc are not part of this card - this is just the presentational aspect